automation/taskcluster/androidTest/ui-test.sh

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env bash
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# This script does the following:
+# 1. Retrieves glcoud service account token
+# 2. Activates gcloud service account
+# 3. Connects to google Firebase (using TestArmada's Flank tool)
+# 4. Executes UI tests
+# 5. Puts test artifacts into the test_artifacts folder
+
+# NOTE:
+# Flank supports sharding across multiple devices at a time, but gcloud API
+# only supports 1 defined APK per test run.
+
+
+# If a command fails then do not proceed and fail this script too.
+set -e
+
+#########################
+# The command line help #
+#########################
+display_help() {
+    echo "Usage: $0 Build_Variant [Number_Shards...]"
+    echo
+    echo "Examples:"
+    echo "To run UI tests on ARM device shard (1 test / shard)"
+    echo "$ ui-test.sh arm -1"
+    echo
+    echo "To run UI tests on X86 device (on 3 shards)"
+    echo "$ ui-test.sh feature x86 3"
+    echo
+}
+
+# Basic parameter check
+if [[ $# -lt 1 ]]; then
+    echo "Your command line contains $# arguments"
```

Comment:
This echo isn't super useful - perhaps we can explicitly say that a build variant was expected, but not provided?

Reply:
you're right, changing it